### ISSUE TITLE
Can we inherit terraform faces from font-lock faces?

### ISSUE THREAD

**Issue by ydzhou** [emacsorphanage/terraform-mode#59]
Hi

At leaset those two faces have hardcoded colors. 

- terraform-resource-type-face
- terraform-resource-name-face

Can we inherit from font-lock faces instead? It does not play well with so many colorschemes and I do not see the point of not having them just using font-lock type or variable faces?

**Comment by rnikoopour** [comment]
They should be configurable in the customization menu for terraform mode.

Which default face would you recommend assigning to each one?

**Comment by ydzhou** [comment]
```
terraform-resource-type-face -> font-lock-type-face))
terraform-resource-name-face -> font-lock-function-name-face))
```

**Comment by Fuco1** [comment]
Yes, I agree.  Inheriting from font-lock faces is the recommended way to setup major mode faces.  If users want some other color scheme, they can then customize the faces separately, but they will always have a "good default" that plays with any theme.

**Comment by ydzhou** [comment]
Yeah. Any plan to update the face to inherit from font-lock? 
I am happy to send a PR to add that.

**Comment by rnikoopour** [comment]
Fixed in #60 

Tag 1.0.1 pushed.  ty for the feedback.

**Comment by Fuco1** [comment]
Thanks @rnikoopour, I was planning to do it over the weekend :blush:
